### PR TITLE
[FEATURE] Quantity input modal

### PR DESCRIPTION
**Pull request by abdullacontractor**
#### Summary
Manual quantity input modal

#### Screenshot
localhost_8100_orders_2_pack iphone 6_7_8 1

### CODE REVIEW

File: addon/components/quantity-input-modal.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import { PropTypes } from 'ember-prop-types';
+import layout from '../templates/components/quantity-input-modal';
+
+const QuantityInputModal = Component.extend({
+  layout,
+  classNames: ['tradegecko-ui-quantity-input-modal'],
+  inputValue: oneWay('inputQuantity'),
+
+  pendingQuantity: computed('inputValue', 'maxQuantity', function() {
+    return this.get('maxQuantity') - this.getWithDefault('inputValue', 0);
```
Comment (selvagsz) [inline addon/components/quantity-input-modal.js]: can be replaced by setting defaultProps to 0

Comment (abdullacontractor) [reply addon/components/quantity-input-modal.js]: Id rather just have this here as well. Doesn't really hurt the code does it?

File: addon/components/quantity-input-modal.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import Component from '@ember/component';
+import { oneWay } from '@ember/object/computed';
+import { computed } from '@ember/object';
+import { PropTypes } from 'ember-prop-types';
+import layout from '../templates/components/quantity-input-modal';
+
+const QuantityInputModal = Component.extend({
+  layout,
+  classNames: ['tradegecko-ui-quantity-input-modal'],
+  inputValue: oneWay('inputQuantity'),
```
Comment (selvagsz) [inline addon/components/quantity-input-modal.js]: any reason to have this ?

File: addon/templates/components/quantity-input-modal.hbs [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{{yield "item-summary"}}
```
Comment (selvagsz) [inline addon/templates/components/quantity-input-modal.hbs]: to reduce verbosity on the consuming end, we can `{{yield (hash itemSummary true)}}` & in consuming end we can
```hbs
{{#quantity-input-modal as |block|}}
  {{#if block.itemSummary}}

  {{/if}}
{{/quantity-input-modal}}
```

**Pull request merged**